ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.12-nightly-x86.yaml

```diff
@@ -22,7 +22,7 @@ resources:
       memory: 200Mi
 tests:
 - as: perfscale-control-plane-5nodes-periodic
-  cron: 29 5 */1 * 1-5
+  cron: 0 9 */14 * 4
```

Comment:
I used a cron expression translator and this would only run the below, is that what we want? Can switch the 4 at the end to allow the tests to run every 14 days no matter which day it is

`At 09:00 on every 14th day-of-month if it's on Thursday`